app/models/book.rb

```diff
@@ -0,0 +1,6 @@
+class Book < ApplicationRecord
+  validates_presence_of :title, :description, :image,
+                        :page_count, :est_reading_time
```

Comment:
Should we validate that the page count is higher than 0 as well?

Reply:
I like that idea, @linda-le1. I think it'll keep us from getting nothing-books being entered